### PR TITLE
bumped the metal client version to 1.5.9

### PR DESCRIPTION
**Pull request by AnuBose**

### CODE REVIEW

**Comment by neilgierman** [comment]
Wondering if we need to update the `version` file as well. @chaitra-mylarappachar ?

**Comment by chaitra-mylarappachar** [comment]
> Wondering if we need to update the `version` file as well. @chaitra-mylarappachar ?

yes, anytime if there is an update to the functionality, we should do this.
This version file is used only when building a standalone provider ( dev purpose, and to host in quake portals ) and not considered when building a published hpegl provider.

**Comment by AnuBose** [comment]
> > Wondering if we need to update the `version` file as well. @chaitra-mylarappachar ?
> 
> yes, anytime if there is an update to the functionality, we should do this. This version file is used only when building a standalone provider ( dev purpose, and to host in quake portals ) and not considered when building a published hpegl provider.

Done.

**Comment by AnuBose** [comment]
@neilgierman , @chaitra-mylarappachar , please review

File: go.mod [hunk 1]
```diff
@@ -6,7 +6,7 @@ require (
 	github.com/golangci/golangci-lint v1.55.2
 	github.com/hashicorp/terraform-plugin-docs v0.18.0
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.33.0
-	github.com/hewlettpackard/hpegl-metal-client v1.5.6
+	github.com/hewlettpackard/hpegl-metal-client v1.5.8
 	github.com/hewlettpackard/hpegl-provider-lib v0.0.14
```
Comment (chaitra-mylarappachar) [inline go.mod:10]: latest version of client is 1.5.9

File: version [hunk 2]
```diff
@@ -1 +1 @@
-1.3.43
+1.3.44
```
Comment (chaitra-mylarappachar) [inline version:1]: it is better to keep the version here match with the repo tag. The latest tag is v1.3.49, so this version should be updated to 1.3.50 ( the next tag that's going to be created).

Comment (neilgierman) [reply version:1]: I just updated the [wiki instructions](https://github.com/hpe-hcss/quake/wiki/Updating-Project-API) to note to use the same version as the next release ta.

**Review by chaitra-mylarappachar — APPROVED** [review]

**Pull request merged**